src/test/ui/editions/edition-keywords-2015-2018-expansion.stderr

```diff
@@ -1,14 +1,18 @@
 error: expected identifier, found keyword `async`
-  --> $DIR/edition-keywords-2015-2018-expansion.rs:8:5
+  --> $DIR/auxiliary/edition-kw-macro-2018.rs:7:19
+   |
+LL |     () => (pub fn async() {})
+   |                   ^^^^^ expected identifier, found keyword
```

Comment:
...for example I think this is clearer.

Reply:
Kind of. On the other hand, in the general case of cross-crate macros (which I believe this is), this span doesn't actually help you any since you can't fix it without an upstream PR...

Reply:
I guess it depends on whether you own the upstream crate or whether you would feel comfortable making said PR. If you would, then it could help?

This is not intentional from my end tho; just a side-effect of removing the hack which I was pointed to :)